aten/src/ATen/Parallel.cpp

```diff
@@ -131,4 +154,32 @@ std::shared_ptr<TaskThreadPoolBase> createC10ThreadPool(
 
 C10_REGISTER_CREATOR(ThreadPoolRegistry, C10, createC10ThreadPool);
 
+void set_num_interop_threads(size_t nthreads) {
+  if (nthreads == 0) {
```

Comment:
why this condition? if you don't want to allow disabling interop thread pool, this should throw (and also <= 0 probably)

Reply:
makes, will upd to throw, it can't be < 0 because it's unsigned